packages/azure/src/application/AzureCredentialsProvider.ts

```diff
@@ -23,6 +26,11 @@ export default class AzureCredentialsProvider {
           clientIdFromGoogle,
           clientSecretFromGoogle,
         )
+      case 'workload_identity':
```

Comment:
To keep this consistent with config naming conventions, this should probably be capitalized as `WORKLOAD_IDENTITY`.